### PR TITLE
feat(OverflowMenu): add 'openUp' prop

### PR DESCRIPTION
**Pull request by cozzbp**

### CODE REVIEW

File: src/components/OverflowMenu/OverflowMenu.js [hunk 1]
```diff
@@ -65,12 +66,12 @@ const OptionsContainer = styled.div`
     position: absolute;
     content: '';
     right: 10px;
-    top: -16px;
-    bottom: auto;
+    top: ${props => props.openUp ? 'auto' : `-16px`};
+    bottom: ${props => props.openUp ? '-16px' : `auto`};
```
Comment (seanevd) [inline src/components/OverflowMenu/OverflowMenu.js]: Did you intend to mix back ticks and quotes?

Comment (cozzbp) [reply src/components/OverflowMenu/OverflowMenu.js]: Nope, I can fix that, though I'm not sure how that works with the weird build process.  Will me committing again break that?

Comment (seanevd) [reply src/components/OverflowMenu/OverflowMenu.js]: It shouldn't break anything. Just Squash and Merge when you merge your PR.

File: src/components/OverflowMenu/OverflowMenu.js [hunk 2]
```diff
@@ -40,7 +40,8 @@ const SelectOption = styled.div`
 
 const OptionsContainer = styled.div`
   position: absolute;
-  top: 31px;
+  top: ${props => props.openUp ? 'initial' : `31px`};
+  bottom: ${props => props.openUp ? '31px' : `initial`};
```
Comment (seanevd) [inline src/components/OverflowMenu/OverflowMenu.js]: I thought you would've caught these back ticks too...

Comment (cozzbp) [reply src/components/OverflowMenu/OverflowMenu.js]: Whoops, fixed.

**Pull request merged**